staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go

```diff
@@ -580,6 +580,35 @@ type PatchOptions struct {
 	FieldManager string `json:"fieldManager,omitempty" protobuf:"bytes,3,name=fieldManager"`
 }
 
+// ApplyOptions may be provided when applying an API object.
```

Comment:
Say something about this being a convenience type for ease of use and actually being the same as PatchOptions?

Reply:
+1. Added. The options fields documentation was all rewritten to focus on apply. I'll call that out too.

Reply:
```
// ApplyOptions may be provided when applying an API object.
// FieldManager is required for apply requests.
// ApplyOptions is equivalent to PatchOptions. It is provided as a convenience with documentation
// that speaks specifically to how the options fields relate to apply.
```